css/style.css

```diff
@@ -39,10 +39,13 @@ button:active, a.button:active {
     background-color: rgba(189, 29, 37, 0.8);
 }
 /* Google stuff */
-.gmnoscreen, .gmnoprint a, .gm-style-cc,
-div a[rel="noopener"] img {
+.gmnoscreen, .gmnoprint a, .gm-style-cc {
     display: none !important;
 }
+div a[rel="noopener"] img {
+    left: 50vw !important;
```

Comment:
Can we use a `left` property corresponding to the actual width of the attribution box?

Reply:
Moved with `10px` space
<img width="230" alt="image" src="https://user-images.githubusercontent.com/23221268/183266971-779770ff-e8f7-444b-adce-cc921ecef97c.png">

Reply:
Can you do it a bit closer, to match the bottom padding?

Reply:
The image's box is now flush with the attribution box, just a little padding in the image itself:
<img width="208" alt="image" src="https://user-images.githubusercontent.com/23221268/183686479-dbff07b9-17b8-47a7-ae6b-056a1e2d85b3.png">